Lib/code.py

```diff
@@ -25,7 +25,7 @@ class InteractiveInterpreter:
     def __init__(self, locals=None):
         """Constructor.
 
-        The optional 'locals' argument specifies the dictionary in
+        The optional 'locals' argument specifies the mapping in
```

Comment:
@ncoghlan , I think this might need double backticks for rST to render as a literal?
``locals``

Reply:
```suggestion
        The optional ``locals`` argument specifies the mapping in
```

Reply:
Since this is a docstring, and `code.py` doesn't otherwise use ReST markup in its docstrings, I decided to leave this alone.